pillar/heroku/discussions.sls

```diff
@@ -181,7 +181,7 @@ heroku:
     OPEN_DISCUSSIONS_BASE_URL: {{ env_data.OPEN_DISCUSSIONS_BASE_URL }}
     OPEN_DISCUSSIONS_COOKIE_DOMAIN: {{ env_data.OPEN_DISCUSSIONS_COOKIE_DOMAIN }}
     OPEN_DISCUSSIONS_COOKIE_NAME: {{ env_data.OPEN_DISCUSSIONS_COOKIE_NAME}}
-    {% if env_data.OCW_NEXT_URL is defined %}
+    {% if env_data.get('OCW_NEXT_URL') is defined %}
```

Comment:
Sorry, wan't clear enough. In this case you can drop the `is defined` and rely on the truthiness of the return value of `.get` which will be `None` if the attribute isn't defined.

Reply:
haha, no worries, I'll make the change now